v20.1/online-schema-changes.md

```diff
@@ -21,6 +21,10 @@ Schema changes consume additional resources, and if they are run when the cluste
 Support for schema changes within [transactions][txns] is [limited](#limitations). We recommend doing schema changes outside transactions where possible. When a schema management tool uses transactions on your behalf, we recommend only doing one schema change operation per transaction.
 {{site.data.alerts.end}}
 
+{{site.data.alerts.callout_info}}
+You cannot start an online schema change if another schema change (e.g., a [primary key change](alter-primary-key.html)) is currently in progress on the same table.
```

Comment:
This isn't quite correct -- you can't start a schema change on a table only if another primary key change is in progress. Other schema changes are ok.

Reply:
Fixed.